packages/core/src/config.js

```diff
@@ -19,8 +19,15 @@ export const schema = {
       },
       requestHeaders: {
         type: 'object',
-        additionalProperties: { type: 'string' },
-        default: {}
+        additionalProperties: { type: 'string' }
+      },
+      authorization: {
+        type: 'object',
+        additionalProperties: false,
+        properties: {
+          username: { type: 'string' },
+          password: { type: 'string' }
+        }
```

Comment:
Another exciting change! We could technically implement this before, but I did it here to bump coverage of the new `Network` class. This allows people to specify authentication credentials to use for basic auth requests. They could also do this with a request header before, but this explicitly allows username/password credentials to be used now.

Reply:
🔥 This is going to make this much easier https://docs.percy.io/docs/capturing-assets-protected-with-authentication